package.json

```diff
@@ -29,22 +29,22 @@
     "@oclif/test": "3.2.15",
```

Comment:
Do we want to throw out node-pty, too?

Reply:
that would probably mean a more impact change, wouldnt it? But i can have a look